### PR TITLE
Comms/GH-3 Redirect Logging

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith gordonjsmith@gmail.com

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq @jbrundage Please Review

File: packages/util/src/logging.ts [hunk 1]
```diff
@@ -58,13 +83,7 @@ export class Logging {
...
             msg = this.stringify(msg);
         }
-        if (isNode) {
-            // tslint:disable-next-line:no-console
-            console.log(`[${n}] ${Level[level].toUpperCase()} ${id}:  ${msg}`);
-        } else {
-            // tslint:disable-next-line:no-console
-            console.log(`[${n}] %c${Level[level].toUpperCase()}%c ${id}:  ${msg}`, `color:${colours[Level[level]]}`, "");
-        }
+        this._writer.write(n, level, id, msg);
```
Comment (miguelvazq) [inline packages/util/src/logging.ts]: Minor: _n_ is a little cloudy in this context. perhaps a more descriptive param?

**Comment by miguelvazq** [comment]
Travis threw an error as well.

**Comment by GordonSmith** [comment]
Changed as per comment and fixed travis issue.

**Pull request merged**